src/types.ts

```diff
@@ -288,8 +290,24 @@ export interface ValueChangedEvent {
   values: ValueChangedEventValues;
 }
 
+export interface TxStateChangedEventStates {
```

Comment:
question) do we need this interface? can we just use before and after in TxStateChangedEvent directly?
if necessary, we can define TransactionStates here as well?

Reply:
I created TxStateChangedEventStates just to implement something similar to the ValueChangedEvent above.
```javascript
export interface ValueChangedEventValues {
  before: any;
  after: any;
}
```
I also think this part is unnecessary.

Reply:
Fixed, thanks!

Reply:
👍 
could you change not to use ValueChangedEventValues either for me? thanks in advance!

Reply:
Fixed!